src/utils/json-data.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs-extra');
+const path = require('path');
+const reserved = require('reserved-words');
+const shared = require('./shared');
+const glob = require('glob');
+const BUILD_TIME_BINDING = '@buildtime:';
+// Matches {{ stache.jsonData. }} with any number of spaces between the '{{' and 'stache.',
+// any keys following the 'jsonData.', and the closing '}}'
+const angularBindingRegExp = new RegExp(/\{\{\s*stache.jsonData.*?\}\}/g);
+const buildTimeBindingRegExp = new RegExp(`\\{\\{\\s*${BUILD_TIME_BINDING}stache.jsonData.*?\\}\\}`, 'g');
+
+let _globalData;
+
+const getGlobalData = () => {
```

Comment:
I disagree. We use one file called `global` for sure,  however the thing over all deals with a stache global data.   No matter what file you put in that folder, it all comes together into one large `global` object.   The name `dataObject` does not make sense, as it literally could refer to anything.   What data?  Why object at all?  `valueString`   It tells you nothing about what it contains.   In this case, as this deals with `stache-data`, the `globalData` could be inferred to mean `stacheGlobalData`  which translates to exactly what it is.  @blackbaud-brandonhare